translations/strings.yaml

```diff
@@ -595,9 +595,13 @@ tipsAndTricks:
   featureTourCardTitle: Tour the VPN
   featureTourCardDescription: Take a tour of Mozilla VPN to learn how the latest features help you protect your online privacy.
   quickTipsTitle: Quick tips
-  quickTipsDescription: Let us help you get the most out of Mozilla VPN
+  quickTipsDescription: Let us help you get the most out of Mozilla VPN.
```

Comment:
You need new string IDs for this type of change, to make sure every locale is updated.

Long explanation available here: https://developer.mozilla.org/en-US/docs/Making_String_Changes

Reply:
Interesting, even if this hasn't made it into a release yet? BTW, the link is a 404.

Reply:
Also: https://transvision.flod.org/string/?entity=vpn_client/mozillavpn.xliff:f8f24fcadf191e23cb6e7479c3f14b61&repo=vpn_client